app/models/user.rb

```diff
@@ -77,9 +77,10 @@ class User < ApplicationRecord
   has_many :tags, dependent: :nullify
   has_many :comments, dependent: :destroy
 
-  scope :not_portus, -> { where.not username: "portus" }
-  scope :enabled,    -> { not_portus.where enabled: true }
-  scope :admins,     -> { not_portus.where enabled: true, admin: true }
+  scope :portus,     -> { where(username: "portus").first }
```

Comment:
Actually declaring `portus` as a scope is a problem because scopes are intended to be chainable and always return a relation instead of the model. I've moved it to a static class method instead. `not_portus` makes sense to be like it is.